File: src/main/java/com/unihogsoft/scalag/vulkan/command/StandardCommandPool.java
package com.unihogsoft.scalag.vulkan.command;

import com.unihogsoft.scalag.vulkan.core.Device;

/**
 * @author MarconZet
 * Created 13.04.2020
 * Copied from Wrap
 */

public class StandardCommandPool extends CommandPool {
    public StandardCommandPool(Device device, Queue queue) {
        super(device, queue);
    }

    @Override
    protected int getFlags() {
        return 0;
    }
}

File: src/main/scala/io/computenode/cyfra/vulkan/command/StandardCommandPool.scala
package io.computenode.cyfra.vulkan.command

import io.computenode.cyfra.vulkan.core.Device

/** @author
  *   MarconZet Created 13.04.2020 Copied from Wrap
  */
class StandardCommandPool(device: Device, queue: Queue) extends CommandPool(device, queue) {
  protected def getFlags: Int = 0
}
